css/main.css

```diff
@@ -396,14 +272,11 @@ header h2 {
 
 .share-icons ul {
     margin:0px;
-
 }
 
 .share-icons li {
-
     display:inline;
     padding:0 0 0 -10px;
```

Comment:
This padding is redundant and not rendering. Be careful about over-applying certain layout elements.